client/lib/js/react/components/asset_detail.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Row, Glyphicon } from 'react-bootstrap/lib';
+import classnames from 'classnames';
+
+
+const AssetDetail = React.createClass({
+    propTypes: {
+        asset: React.PropTypes.object,
+        assetContent: React.PropTypes.string,
+        children: React.PropTypes.oneOfType([
+            React.PropTypes.arrayOf(React.PropTypes.node),
+            React.PropTypes.node
+        ]),
+        className: React.PropTypes.string
+    },
+
+    getDefaultProps() {
+        return {
+            className: 'pull-right'
+        };
+    },
+
+    getAssetContent() {
+        const { asset, assetContent } = this.props;
+        if (!assetContent) {
+            return asset.transaction.data ? asset.transaction.data.payload.content : '-';
```

Comment:
Yeah totally fine, but we might want to check if the data as we expect it to be, and if so continue with code execution (meaning putting an if-statement in front or initial schema validation).